create_bfb

```diff
@@ -257,15 +257,15 @@ copy_rpm()
 }
 
 # Copy the content of RPM packages
-for package in  mlxbf-bfscripts util-linux dosfstools e2fsprogs kmod glibc coreutils pciutils usbutils chkconfig mft mft-oem kernel-mft tar which
+for package in  mlxbf-bfscripts util-linux dosfstools e2fsprogs kmod glibc coreutils pciutils usbutils chkconfig mft mft-oem kernel-mft tar which curl openssl $ADDON_RPMS
```

Comment:
Where is this defined?

Reply:
I think is added as an option to add new RPMs is not used as far as i know
Change is also coming from the Nvidia/Mariner repo